src/CodeAnalysis.ruleset

```diff
@@ -72,6 +72,149 @@
     <Rule Id="CA5371" Action="None" /> <!-- Use XmlReader For Schema Read -->
     <Rule Id="CA5372" Action="None" /> <!-- Use XmlReader For XPathDocument -->
   </Rules>
+  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
+    <Rule Id="AD0001" Action="None" /> <!-- Analyzer threw an exception -->
```

Comment:
📝 This should be enabled

Reply:
> This should be enabled

It fails.  StyleCop encountered null reference exceptions in at least one case.

Reply:
@sharwell, since I just happened to see one while enabling some more rules, here's one example failure:
```
Analyzer 'StyleCop.Analyzers.ReadabilityRules.SA1139UseLiteralSuffixNotationInsteadOfCasting' threw an exception of type 'System.NullReferenceException' with message 'Object reference not set to an instance of an object.'. [D:\repos\corefx\src\System.Data.Common\src\System.Data.Common.csproj]
```